app/src/main/java/com/framgia/wsm/screen/listrequest/ItemListRequestViewModel.java

```diff
@@ -128,18 +127,26 @@ public String getLeaveTypeCode() {
     }
 
     public String getFromTime() {
-        if (mRequest != null) {
+        if (mRequest != null && mRequest.getCompensation() != null) {
             return mRequest.getCompensation().getFromTime();
         }
         if (mRequestOverTime != null) {
             return mRequestOverTime.getFromTime();
         }
         if (mRequestOff != null) {
-            if(!mRequestOff.getStartDayHaveSalary().getOffPaidFrom().equals("")){
-                return mRequestOff.getStartDayHaveSalary().getOffPaidFrom();
+            if (mRequestOff.getStartDayHaveSalary() != null) {
+                if (!mRequestOff.getStartDayHaveSalary().getOffPaidFrom().equals("")) {
```

Comment:
WARNING: [CollapsibleIfStatements] These nested if statements could be combined
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/basic.html#CollapsibleIfStatements